samples/scripts/syncAnnotationsScript.py

```diff
@@ -0,0 +1,261 @@
+def clone_and_upload_annotations(
+    gc: girder_client.GirderClient, annotations, rootFolder
+):
+    for annotation in annotations:
+        upload_folder = annotation["girderId"]
+        upload_file = annotation["file"]
+        name = f"{os.path.basename(annotation['file']).replace('.csv','')}"
+        if annotation["label"] != default_label:  # now we change the name
+            name = f"{name} - {annotation['label']}"
+        new_folder = gc.sendRestRequest(
+            "POST",
+            f'dive_dataset?cloneId={annotation["girderId"]}&parentFolderId={rootFolder}&name={name}',
+        )
+        upload_folder = str(new_folder["_id"])
+        gc.uploadFileToFolder(upload_folder, upload_file)
+        gc.sendRestRequest(
+            "POST",
+            f"dive_rpc/postprocess/{upload_folder}",
+            data={"skipTranscoding": True, "skipJobs": True},
+        )
+
+
+# returns the public folder of the user who logged in
+def get_public_folder(gc: girder_client.GirderClient):
+    current_user = gc.sendRestRequest("GET", "user/me")
+    userId = current_user["_id"]
+    folders = gc.sendRestRequest(
+        "GET",
+        f"folder?parentType=user&parentId={userId}&text=Public&limit=50&sort=lowerName&sortdir=1",
+    )
+    if len(folders) > 0:
+        uploadFolder = folders[0]["_id"]
+    else:
+        print("No folder found for the user")
+    return uploadFolder
+
+
+def ask_question(prompt):
+    while True:
+        response = input(prompt).strip().lower()
+        return response
+
+
+def ask_yes_no_question(prompt):
+    while True:
+        response = input(prompt).strip().lower()
+        if response == "y":
+            return True
+        elif response == "n":
+            return False
+        else:
+            print("Invalid input. Please enter 'y' or 'n'.")
+
+
+@click.command(name="LoadData", help="Load in ")
+@click.argument(
+    "folder"
+)  # a local folder to search for mp4 video files and json/csv files.
+@click.argument("girder_id", required=False)
+def load_data(folder, girder_id):
+    baseGirderId = girder_id
+    annotations = get_annotations(folder)
+    print(annotations)
+    gc = login()
+    base_path = getBasePath(gc, baseGirderId, baseGirderType)
+    print(base_path)
+    upload_list = []
+    for annotation in annotations:
+        print(annotation)
+        result = find_dataset(gc, annotation, base_path)
+        if result:
+            upload_list.append(
+                {
+                    "girderId": result["id"],
+                    "label": result["label"],
+                    "parentId": result["parentId"],
+                    "file": annotation,
+                    "type": "upload",
+                }
+            )
+    local_public = ask_yes_no_question(
+        f'Would you like to create a cloned copy of the images in your public folder?  If you choose "n" it will upload the annotations to the source image locations. (y/n)\n'
```

Comment:
```suggestion
        f'Would you like to create a cloned copy of the images in your DIVE public folder?  If you choose "n" it will upload the annotations to the source image locations. (y/n)\n'
```